plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/HiveTableOperations.java

```diff
@@ -255,7 +255,7 @@ public String metadataFileLocation(String filename)
     public LocationProvider locationProvider()
     {
         TableMetadata metadata = current();
-        return LocationProviders.locationsFor(metadata.location(), metadata.properties());
+        return getLocationProvider(metadata.location(), metadata.properties());
```

Comment:
nit: capitalize `use Iceberg LocationProvider instead of hardcoded path` commit message